graph/types/discussion.graphqls

```diff
@@ -30,6 +30,8 @@ type Discussion implements Entity {
     idleMinutes: Int!
     tags: [Tag!]
     upcomingContent: [ImportedContent!]
+    flairTemplates: [FlairTemplate!]
+    accessRequests: [DiscussionAccessRequest!]
```

Comment:
I'm thinking we start to make the (annoying) move to make these connections pretty soon.

Reply:
https://github.com/nedrocks/delphisbe/issues/34